app/src/renderer/components/transactions/LiBankTransaction.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <tm-li-transaction
-    :color="color"
+  <li-transaction
+    :color="`#ED553B`"
```

Comment:
this is a bit odd

Reply:
why ?

Reply:
if it's not dynamic it shouldn't need to be a prop

Reply:
it is dynamic, it is different on every transaction category
But I would also collect all the colors somewhere or color the circles via classes.